tests/parse_tests/file_parsing_tests/file_parsing_test.py

```diff
@@ -17,3 +17,9 @@ def test_complex_file(self):
         parser = Parser()
         result = parser.parse(file)
         self.assertEqual("Success", result["Status"])
+
+    def test_if_inside_function(self):
+        file = open(TESTS_RESOURCES_PATH + "/files_to_parse/if_stmt_inside_function.brs", "r+").read()
```

Comment:
May be possible to use `interface_handler -> file_reader(file_to_lex)` (and move the method to utilities). Have discovered a bigger issue related to this though - will discuss on slack.